src/dos/program_imgmount.cpp

```diff
@@ -212,13 +212,20 @@ void IMGMOUNT::Run(void) {
         return;
     }
 
+    std::vector<std::string> files;
     // find all file parameters, assuming that all option parameters have been removed
     while (cmd->FindCommand((unsigned int)(paths.size() + 2), temp_line) && temp_line.size()) {
         // Try to find the path on native filesystem first
         const std::string real_path = to_native_path(temp_line);
         if (real_path.empty()) {
-            LOG_MSG("IMGMOUNT: Path '%s' not found, maybe it's a DOS path",
-                    temp_line.c_str());
+            if (get_expanded_files(temp_line, files, true)) {
+                paths.insert(paths.end(), files.begin(), files.end());
```

Comment:
Suggest drop the temporary "files" `vector` and passing  `paths` into `get_expanded_files` instead.
(more comments when you get to `get_expanded_files` below)

Reply:
Thanks for the suggestion! Done.